extras/uwsgi/uwsgi.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=uWSGI Emperor
-
-[Service]
-ExecStart=/usr/bin/uwsgi --emperor /etc/uwsgi/apps-enabled --logto /var/log/uwsgi/access.log
-RuntimeDirectory=uwsgi
-Restart=always
-KillSignal=SIGQUIT
-Type=notify
-NotifyAccess=all
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
Yes, Debian already provides a sysvinit script.

You can have services use `After=uwsgi.service` even if uwsgi is in fact started by a sysvinit script.

And if you want uwsgi to start after mongodb, then you can continue doing either of these:

1. drop in a `mongodb.service.d/whatever.conf` file that simply adds the required `Before=uwsgi.service` line
2. drop in a `uwsgi.service.d/whatever.conf` file that simply adds the required `After=mongodb.service` line

And it will just work.

I'm not sure why you would want any ordering between mongodb and uwsgi, since AFAIK ideascube doesn't use mongodb, but you can, there's no issue with that. However, this isn't ideascube responsibility IMHO, so you just have to do it yourself.